src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/court/libra/LibraCourtCaseListenerIntTest.kt

```diff
@@ -21,33 +21,34 @@ import uk.gov.justice.digital.hmpps.personrecord.model.identifiers.PNCIdentifier
 import uk.gov.justice.digital.hmpps.personrecord.model.person.Address
 import uk.gov.justice.digital.hmpps.personrecord.model.person.Person
 import uk.gov.justice.digital.hmpps.personrecord.model.types.SourceSystemType
-import uk.gov.justice.digital.hmpps.personrecord.service.type.TelemetryEventType
+import uk.gov.justice.digital.hmpps.personrecord.model.types.SourceSystemType.LIBRA
 import uk.gov.justice.digital.hmpps.personrecord.service.type.TelemetryEventType.COURT_MESSAGE_RECEIVED
+import uk.gov.justice.digital.hmpps.personrecord.service.type.TelemetryEventType.CPR_CANDIDATE_RECORD_SEARCH
 import uk.gov.justice.digital.hmpps.personrecord.service.type.TelemetryEventType.CPR_RECORD_CREATED
 import uk.gov.justice.digital.hmpps.personrecord.test.messages.libraHearing
 import uk.gov.justice.hmpps.sqs.countMessagesOnQueue
 import java.time.LocalDate
+import java.util.UUID
 import java.util.concurrent.TimeUnit.SECONDS
 
 class LibraCourtCaseListenerIntTest : MessagingMultiNodeTestBase() {
 
   @BeforeEach
   override fun beforeEach() {
-    super.beforeEach()
-    personRepository.deleteAll()
```

Comment:
As I'm going to be working locally trying to do the cluster mapping, I wanted to see if I could stop the person table from being emptied by tests as it will slow me down.